src/index.html

```diff
@@ -12,6 +12,178 @@
   </head>
   <body>
     <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" onsubmit="onSubmit()">
+      <fieldset>
+        <legend>Personal information</legend>
+        <div class="item">
+          <label for = "surname">
+                Surname:
+          </label>
```

Comment:
```suggestion
          <label for="surname">
            Surname:
          </label>
```
Два отступа достаточно